File: Models/Requests/Login/RequestLogin.cs
using System.ComponentModel.DataAnnotations;

namespace Models.Requests.Login;

/// <summary>
/// 로그인 요청 정보
/// </summary>
public class RequestLogin : RequestBase
{
    /// <summary>
    /// 로그인 아이디
    /// </summary>
    [Required(ErrorMessage = "아이디를 입력해주세요")]
    [MinLength(1 , ErrorMessage = "아이디를 입력해주세요")]
    public string LoginId { get; set; } = "";

    /// <summary>
    /// 로그인 아이디
    /// </summary>
    [Required(ErrorMessage = "패스워드를 입력해주세요")]
    [MinLength(1 , ErrorMessage = "패스워드를 입력해주세요")]
    public string Password { get; set; } = "";

}

File: BudgetBringer/src/models/requests/login/request-login.ts
/**
 * 로그인 요청 모델 클래스
 */
export class RequestLogin {
  /**
   * 로그인 아이디
   */
  public loginId : string = '';

  /**
   * 패스워드
   */
  public password : string = '';

  /**
   * 생성자
   * @param loginId 로그인 아이디
   * @param password 패스워드
   */
  constructor(loginId: string, password: string) {
    this.loginId = loginId;
    this.password = password;
  }
}
